stdgo/reflect/Reflect.hx

```diff
@@ -943,12 +1133,65 @@ class Value {
 			case GT_string:
 				return (val : GoString);
 			case _:
-				throw "Value.String not implemented for " + underlyingType.string();
+				throw new ValueError("String", underlyingType.kind());
 		}
 		return "";
 	}
 }
 
+// A ValueError occurs when a Value method is invoked on
+// a Value that does not support it. Such cases are documented
+// in the description of each method.
+@:rtti class ValueError extends StdGo2hx {
```

Comment:
Needs to implement interface stdgo.StdGoTypes.Error

Reply:
Will do.